Description: Fix timeit when used with Python 2.7.11
 Causes many tests to fail otherwise.
Origin: https://git.archlinux.org/svntogit/community.git/tree/trunk?h=packages/sagemath
Bug: https://trac.sagemath.org/ticket/19735

--- a/sage/src/sage/misc/sage_timeit.py
+++ b/sage/src/sage/misc/sage_timeit.py
@@ -217,7 +217,7 @@
     # but is there a better way to achieve that the code stmt has access
     # to the shell namespace?
 
-    src = timeit_.template % {'stmt': timeit_.reindent(stmt, 8),
+    src = timeit_.template % {'init': "", 'stmt': timeit_.reindent(stmt, 8),
                              'setup': "pass"}
     code = compile(src, "<magic-timeit>", "exec")
     ns = {}
